ReactSkia/JSITurboModuleManager.cpp

```diff
@@ -15,6 +14,10 @@
 #include "modules/RSkTVNavigationEventEmitter.h"
 #include "utils/RnsLog.h"
 
+#if ENABLE(FEATURE_ALERT)
```

Comment:
ENABLE needs utils.h to be included before this. So moving "RSkAlertManager.h"" it to the end, to avoid compilation error.